storybook/storybook-react/config/preview.tsx

```diff
@@ -21,5 +61,9 @@ export const parameters = {
       order: ['Buttons', 'Containers', 'Feedback', 'Forms', 'Layout', 'Media', 'Navigation', 'Text'],
     },
   },
+  viewport: {
+    defaultViewport: 'responsive',
+    viewports,
+  },
```

Comment:
This doesn’t seem to work.

Reply:
If I read the docs then it makes sense, the defaultViewport already is set to 'reponsive'.

Maybe you can test it with the `INITIAL_VIEWPORTS` example from [the docs](https://storybook.js.org/docs/react/essentials/viewport)?